src/sentry/integrations/discord/commands.py

```diff
@@ -32,7 +32,7 @@ class DiscordCommandManager:
         },
     ]
 
-    def register_commands(self) -> None:
+    def register_commands(self, client: DiscordNonProxyClient) -> None:
```

Comment:
the default for the `client` is a class, seems like the code expects an instance